src/ts/components/Container.ts

```diff
@@ -129,6 +129,10 @@ export class Container<Config extends ContainerConfig> extends Component<Config>
       'aria-label': i18n.performLocalization(this.config.ariaLabel),
     }, this);
 
+    if (typeof this.config.tabIndex === 'number') {
```

Comment:
The `ComponentConfig` already has the tabIndex property but it's not consistently applied when creating the dom elements.